### ISSUE TITLE
Vanilla clients without mods can go into the mod Rift server

### ISSUE THREAD

**Issue by Mgazul** [DimensionalDevelopment/Rift#23]
Rift-1.0.3-50

Vanilla clients without mods can go into the mod Rift server
No errors, just saw some hidden blocks<mod blocks>

**Comment by Runemoro** [comment]
Yes, I agree there should be a system for checking whether mods are installed on both the server and client when connecting, but it shouldn't be based on the mod side. A mod may work on both sides without requiring it to be installed on both sides. This is common for optimization mods with optimizations for both sides, and for server mods with optional client support.
